src/components/Admin/ProductList.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import ProductItem from "./ProductItem";
+
+const ProductList = ({ productList, editProductSubmit, deleteProduct }) => {
+  let products = productList;
```

Comment:
looks useless to me, if afraid of mutation by ref use distruction `[...productList].map`